lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -18,13 +33,32 @@ def self.run(params)
           violations.each do |lang, diff|
             UI.error "Inconsistencies found between '#{params[:base_lang]}' and '#{lang}':\n\n#{diff}\n"
           end
-          if params[:abort_on_violations] && !violations.empty?
-            UI.abort_with_message!('Inconsistencies found during Localization linting. Aborting.')
-          end
-          
+
           violations
         end
 
+        RETRY_MESSAGE = <<~MSG
+          Inconsistencies found during Localization linting.
+          You need to fix them before continuing. From this point on, you should:
+
+          - Either cancel this lane (reply 'No' below), then work with polyglots in #i18n
+            to fix those directly in GlotPress – by rejecting the inconsistent
+            translations, or by submitting a fixed copy. Then run the lane again.
```

Comment:
Nitpick

```suggestion
            translations, or by submitting a fixed copy. Rerun the lane when everything
            has been fixed.
```

Imho this sounds better than the repeated then (then work... then run...).